Addressing feedback (#29)
HorizenLabs/zkverify-docs: docs/tutorials/05-how_to_run_a_node/02-run_using_docker/04-run_validator_node.md

![alt_text](./img/polkadotjs_staking_validate.png)

Insert your account password and confirm by clicking on button `Sign and Submit`. Wait for pop-up message confirming successful submission. As an additional double check you can navigate to the section `Network` then to the subsection `Staking`, click on `Active` tab and verify that your validator is within the list.

## Conclusion

That's it! You just need to wait for the completion of the current era and the next one (since an era lasts for 6 hours, in the worst case this would be 12 hours).  After that your node will start authoring new blocks. You can check this by navigating to the section `Network` then to the subsection `Explorer` for a summarized view of the list of recently authored blocks, or to the section `Network` then to the subsection `Staking` for an advanced console specifically designed for staking. On the Docker side you can check the logs and expect messages like the following one:

![alt_text](./img/node_authoring.png)

> reporting your validator node is nomore just syncing the blockchain but actually contributing by authoring new blocks (`Starting consensus session...` and `Pre-sealed block for proposal...`).

Reviewer: ```suggestion
reporting your validator node is not only syncing the blockchain but also contributing by authoring new blocks (`Starting consensus session...` and `Pre-sealed block for proposal...`).
```

Author: Ok addressed as per your suggestion.
